docs/60_initialize_vendors.md

```diff
@@ -0,0 +1,448 @@
+
+These values can also be replicated in the settings. 
+
+<br/><br/>
+<img
+src="../img/netlify_config.png"
+alt="retrieve"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+Your frontend should be able to deploy to an auto-generated URL via Netlify now! Steps for domain management and configuration are described further in the [Frontend Deployment](../64_dandi_archive/#frontend-deployment) section of these within the DANDI Archive setup.
+
+## Sentry
+
+[Sentry](https://sentry.io/) is a monitoring tool used for the DANDI Archive API. It is integral in order to notify engineers if a system is down, experiencing poor performance, or may have unwanted users
+
+Begin by creating a Sentry account -- once successful, you'll start by creating a new Project:
+
+<br/><br/>
+<img
+src="../img/sentry_init.png"
+alt="retrieve"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+##### Select Django as an App Type
+
+DANDI Archive API is built as a Django app -- so proceed to select `Django` on the following screen:
+
+<br/><br/>
+<img
+src="../img/sentry_django.png"
+alt="retrieve"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+##### Capture Sentry DSN value
+
+You'll be provided with a screen displaying how to initialize and install Sentry into your Django app. For now, just capture the DSN value.
+This value will be used later as an environment variable while deploying your API via Terraform.
+
+<br/><br/>
+<img
+src="../img/sentry_sdk_dsn.png"
+alt="retrieve"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+## PyPI
+
+A common way that users interact with the DANDI ecosystem is via its CLI and Python client tool: [dandi-cli](https://pypi.org/project/dandi/)
+
+##### Create a PyPI Account
+
+Versions of this tool are hosted, versioned and managed by PyPI -- therefore, you will need to create an account.
+
+##### Retrieve an API Token
+
+After creating an account, you'll want to create your own API key -- first go to `Account settings`
+
+<br/><br/>
+<img
+src="../img/pypi_account.png"
+alt="retrieve"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+Scroll down to `API tokens` and create a value -- keep track of this value for now. We will use it later
+
+<br/><br/>
+<img
+src="../img/pypi_api_token.png"
+alt="retrieve"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+In terms of creating and publishing your first project, go to the [documentation for setting up a CLI and Python Client](.
+./47_cli_python)
+
+
+## datalad (TBD)
+
+## git-annex (TBD)
```

Comment:
@yarikoptic needs to finish up but likely elsewhere and here just describe setup of the box:

```suggestion
## A host for extra services

Some services are not yet integrated within the main infrastructure:

- https://github.com/dandi/backups2datalad - to populate/update https://github.com/dandi/dandisets, https://github.com/dandisets, and https://github.com/dandizarrs/
- TODO: heroku logs
- TODO: aws s3 access stats dump
- TODO: con/tinuous dumps of CI logs
- TODO: zarr manifests generation (ATM not on drogon even)
- TODO: access stats analysis/plots (yet to be finished/cron deployed)
```

Reply:
Thanks Yarik -- updated